packages/docusaurus-remark-plugin-import-file/src/index.ts

```diff
@@ -3,7 +3,8 @@ const { resolve } = require("path");
 const visit = require("unist-util-visit");
 
 // inspired from https://github.com/dotansimha/remark-import-partial
-module.exports = function () {
+export default function () {
+    // @ts-ignore
     const unified: any = this;
```

Comment:
I wasn't absolutely sure what the approach could/should be here for this remark plugin, as I'm not sure who binds the `this` as `unified` (maybe Remark does?).

Reply:
copy paste voodoo

Reply:
Is there a good way to validate I haven't regressed this with this change? I assume build, link, etc. Just not sure what to link it to...

Reply:
you should be able to run the jest test suite from that plugin folder

Reply:
the plugin is in action here https://microsoft.github.io/docusaurus-plugins/docs/markdown-features/import-file

Reply:
Ah cool. So long as it's covered in tests we should be GTG!